consultation_analyser/consultations/models.py

```diff
@@ -83,16 +82,9 @@ class Theme(UUIDPrimaryKeyModel, TimeStampedModel):
 
```

Comment:
is_outlier is no longer calculated (at least not in the same way!)

Reply:
Doh! Have updated the editable status and also the comment!